lingui.config.ts

```diff
@@ -1,11 +1,11 @@
 import type { LinguiConfig } from '@lingui/conf';
 
 const config: LinguiConfig = {
-  locales: ['en', 'ru', 'es'],
+  locales: ['en', 'ru', 'es', 'de', 'lt', 'ta', 'pt_BR'],
   formatOptions: { lineNumbers: false },
   catalogs: [
     {
-      include: ['apps/postybirb-ui/src', 'libs/translations/src'],
+      include: ['apps/postybirb-ui/src/', 'libs/translations/src/'],
```

Comment:
for some reasom it was not picking up directories when i run extract on my phone without these trailling slashes